test.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebSite</title>
+    <link rel="stylesheet" href="test.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+  <body>
+      <header class="top-menu">
+        <nav>
+            <ul>
+                <li>
+                    <a>Navigation</a>
+                </li>
+                <li>
+                    <a>psd websites</a>
+                </li>
+                <li>
+                    <a>ui kits</a>
+                </li>
+                <li>
+                    <a>premium psd files</a>
+                </li>
+                <li>
+                    <a>about</a>
+                </li>
+                <li>
+                    <a>blog</a>
+                </li>
+            </ul>
+        </nav>    
+        <div class="navigation-menu">
+            <i class="fas fa-bars"></i>
+        </div>
+      </header>
+      <section class="awesome-PSD">
+        <h1>Awesome Free PSD templates & Resources</h1>
+        <div class="link-Arrow">
+             <a>www.BlazRobar.com</a>
+             <i class="fas fa-angle-right"></i>
+         </div>
+      </section>
+      <section class="robinHood-PSD">
+            <article class="robinHood-PSD__left-section">
```

Comment:
I assume according to BEM it would be better:
.robin-hood-psd__section {
//common styles for both left and right sections
}

.robin-hood-psd__section--left {
//styles for the left section
}

.robin-hood-psd__section--right {
//styles for the right section
}